aws-copilot.hcl

```diff
@@ -0,0 +1,48 @@
+description = "AWS Copilot is an open source command line interface that makes it easy for developers to build, release, and operate production ready containerized applications on AWS App Runner, Amazon ECS, and AWS Fargate."
+test = "copilot --help"
+binaries = ["copilot"]
+
+darwin {
+    source = "https://github.com/aws/copilot-cli/releases/download/v${version}/copilot-darwin"
```

Comment:
Ah I think what's happened is that they have added support for more architectures over time, but keep the old filenames for compatibility.

I think you can actually DRY it further with this:

```hcl
linux {
  source = "https://github.com/aws/copilot-cli/releases/download/v${version}/copilot-${os}-${arch}"
  on unpack {
    rename { from = "${root}/copilot-${os}-${arch}" to = "${root}/copilot" }
  }
}

darwin {
  source = "https://github.com/aws/copilot-cli/releases/download/v${version}/copilot-darwin"
  on unpack {
    rename { from = "${root}/copilot-darwin" to = "${root}/copilot" }
  }
}
```

Reply:
looking much more concise now 👍🏼